src/button/index.js

```diff
@@ -1 +1,2 @@
-export default from './Button';
+export default from './StatefulButton';
+export PureButton from './Button';
```

Comment:
if `StatefulButton` is the default maybe we should rename `Button` to `PureButton` and `StatefulButton` to `Button`:
I find confusing that `import Button from '../button';` will import `StatefulButton` instead of `Button`